### PR TITLE
feat: :sparkles: Mumbai is joining the party 😀

### PR DESCRIPTION
**Pull request by indcoder**
1. Added the community | 2. Furnished requested event details

### CODE REVIEW

File: 2022/globalazuremumbai/data.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      "indcoder",
+      "",
+      ""
+    ]
+  },
+  "CommunityInfo": {
+    "Name": "Global Azure Bootcamp 2022 Mumbai",
+    "Country": "India",
+    "Url": "https://hackmum.in/",
+    "SessionizeApiId": "https://sessionize.com/gab22mum/",
```
Comment (noopman) [inline 2022/globalazuremumbai/data.json:14]: This is not the event sessionizeid, right @davidjrh!

**Review by noopman — COMMENTED** [review]
We need to ensure it is the right sessionizeid in the metadata.

File: 2022/globalazuremumbai/data.json (hunk 1 above)
Comment (indcoder) [reply 2022/globalazuremumbai/data.json:14]: Yes, my bad. I had inadvertently put the CfP URl. Apologies

Comment (indcoder) [reply 2022/globalazuremumbai/data.json]: @noopman SessionizeAPI  has been updated with the API code

Comment (davidjrh) [reply 2022/globalazuremumbai/data.json]: @indcoder @noopman yes, now it is fine. Merged!

**Pull request merged**